### PR TITLE
Fix: Documentation Bug

### PR DESCRIPTION
**Pull request by HalphasX**
Fixes : 

- [x] Remove colons from each command
- [ ] Fix headers that needs referencing (unsure, still need to upload to github to see)
- [x] Remove information_source from the box description
- [x] Restructure UG to match (FAQ to below, and command summary in one place)
- [ ] #204 

Still coming up : 

- [ ] Screenshots (already ssed, only need to update)
- [ ] Help page will update to support command summary
- [ ] Reword the notes in the command summary and some command lists to use tips, cautions, and information box
- [ ] CS2101: update to adhere to current target audiencce

Do tell if there is any additional change to be made.

### CODE REVIEW

**Comment by EltonGohJH** [comment]
UI.png is needed for the team product page.

File: docs/UserGuide.md [hunk 1]
```diff
@@ -534,37 +551,28 @@ Notor has keyboard shortcuts to give you added convenience and allow you to cont
...
-  the card.
-* For the **List** commands, to view list of persons in a particular subgroup, you should first use **List** command to list out all the subgroups in a `Group` you want to view,
-  then list out the persons inside that subgroup by **List** command with the `GROUP_INDEX` you want to list out.
-
-## FAQ
-
-**Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains
-the data of your previous Notor home folder.
+Refer to the details of Person commands [here](#person).
```
Comment (EltonGohJH) [inline docs/UserGuide.md:575]: This goes to here literally. It goes to the summary. I think it is suppose to go up to the commands.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -590,7 +598,20 @@ Action                    | Format
...
+
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+* For the **Create** and **List** commands, if you want to automatically add them to a group, please use the **List** command to make sure the `Group` you want to
+  use the command on is displayed before using them via `GROUP_INDEX`.
+* For the **List** commands, to view list of persons in a particular subgroup, you should first use **List** command to list out all the subgroups in a `Group` you want to view,
+  then list out the persons inside that subgroup by **List** command with the `GROUP_INDEX` you want to list out.
+</div>
+
 ### Group
+Refer to the details of the group commands [here](#group).
```
Comment (EltonGohJH) [inline docs/UserGuide.md:614]: This goes to here literally. It goes to the summary. I think it is suppose to go up to the commands.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -611,10 +632,17 @@ for the **List** command, the `INDEX` argument can be either a `Group` or a `Per
 </div>
 
 ### General
+Refer to the details of the general commands [here](#general).
```
Comment (EltonGohJH) [inline docs/UserGuide.md:635]: This goes to here literally. It goes to the summary. I think it is suppose to go up to the commands.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -101,30 +102,32 @@ If you type in a command with no arguments and the command requires arguments, t
...
-#### _Exiting the program_ :
+#### _Exiting the program_
 
 Exits the program and saves your data.
 
 Format: `exit`
 
-####<a name = "exporting-data"></a> _Exporting data_ :
+<a name = "exporting-data"></a> 
+####_Exporting data_
```
Comment (EltonGohJH) [inline docs/UserGuide.md:120]: There is a bug with this due to the lack of space between #### and _

File: docs/UserGuide.md (hunk 3 above)
Comment (HalphasX) [reply docs/UserGuide.md]: fixed, please check

File: docs/UserGuide.md [hunk 5]
```diff
@@ -534,37 +551,28 @@ Notor has keyboard shortcuts to give you added convenience and allow you to cont
...
-
 Shortcut Key            | Linux/Window                                   | MacOS
 ------------------------|------------------------------------------------|---------------------------------------
 **TimeStamp**           | `Ctrl + T`                                     | `Cmd + T`
 **Save**                | `Ctrl + S`                                     | `Cmd + S`
 **Quit without Saving** | `Ctrl + W`                                     | `Cmd + W`
 **Save and Quit**       | `Ctrl + Q`                                     | `Cmd + Q`
 
+<div markdown="block" class="alert alert-info">
+**Notes on Timestamp:**<br>
```
Comment (EltonGohJH) [inline docs/UserGuide.md:562]: Bug here

File: docs/UserGuide.md [hunk 6]
```diff
@@ -590,7 +598,20 @@ Action                    | Format
 **List Archived Persons**       | `person /listarchive`                                                           | `p /lar`
 **Unarchive**             | `person (INDEX) /unarchive`                                                   | `p (INDEX) /uar`
 
+<div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
```
Comment (EltonGohJH) [inline docs/UserGuide.md:601]: Bug here also

File: docs/UserGuide.md [hunk 7]
```diff
@@ -590,7 +598,20 @@ Action                    | Format
...
+* For the **Add** and **Remove** commands, please make sure that the `GROUP_NAME` is typed exactly as how it is spelt on
+  the card.
+</div>
+
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
```
Comment (EltonGohJH) [inline docs/UserGuide.md:606]: Bug also

File: docs/UserGuide.md (hunk 4 above)
Comment (EltonGohJH) [reply docs/UserGuide.md:120]: Not fixed yet

**Review by EltonGohJH — COMMENTED** [review]
image

All the tips and caution are bugged. I think they are supposed to be bold lol

File: docs/UserGuide.md (hunk 4 above)
Comment (HalphasX) [reply docs/UserGuide.md:120]: got it

**Pull request closed without merging**